libllvm-pdb-wrapper/wrapper.cpp

```diff
@@ -333,6 +368,9 @@ EXPORT void PDB_File_Destroy(void *Instance) {
 
 EXPORT int PDB_File_Commit(void *Instance, const char *InputPath, const char *OutputPath) {
     auto pdb = (pdb_file *) Instance;
+#ifdef NEWER_LLVM
+    pdb->finalize_public_symbols();
```

Comment:
this may or may not be the correct place for it but it was needed to commit the bulk symbols to the pdb.